components/script/webdriver_handlers.rs

```diff
@@ -356,6 +356,35 @@ pub fn handle_add_cookie(
         .unwrap();
 }
 
+pub fn handle_delete_cookies(
+    documents: &Documents,
+    pipeline: PipelineId,
+    reply: IpcSender<Result<(), WebDriverCookieError>>,
+) {
+    let document = match documents.find_document(pipeline) {
```

Comment:
@jdm Dont we select the document by using a pipeline id?

Reply:
We do, but there is only one resource thread that is shared between all documents so the message always goes to the same place.